src/components/main-view/main-view.jsx

```diff
@@ -44,75 +51,195 @@ export class MainView extends React.Component {
     });
   }
 
-  onLoggedIn(user) {
+  onLoggedIn(authData) {
     this.setState({
-      user,
+      user: authData.user.Username,
     });
+
+    localStorage.setItem("token", authData.token);
+    localStorage.setItem("user", authData.user.Username);
+    this.getMovies(authData.token);
   }
 
-  logOut() {
+  getMovies(token) {
+    axios
+      .get(`${process.env.MY_FLIX_API}/movies`, {
+        headers: { Authorization: `Bearer ${token}` },
+      })
+      .then((response) => {
+        this.setState({
+          movies: response.data,
+        });
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+  }
+
+  updateUser(newUserInfo) {
+    const { userName, newPassword, email } = newUserInfo;
+    axios
+      .put(
+        `${process.env.MY_FLIX_API}/users/${currentUser}`,
+        {
+          Username: userName,
+          Password: newPassword,
+          Email: email,
+        },
+        {
+          headers: { Authorization: `Bearer ${token}` },
+        }
+      )
+      .then((response) => {
+        console.log(response);

```

Comment:
You would have to add a popup or toast when it done updating the user.

please check this example of using toast in bootstrap:
https://react-bootstrap.github.io/components/toasts/